package.json

```diff
@@ -19,6 +19,7 @@
     "node": "^20.9.0"
   },
   "dependencies": {
+    "@types/google.accounts": "^0.0.15",
```

Comment:
nit: can't really figure out what we need this for, but it also seems like the simplest package to pull in 🤔

Reply:
Yeah, this adds the necessary types for the FE `window.google.accounts` typing errors

Reply:
I am confused, I don't really see any types being used around that. 🤔